tsconfig.json

```diff
@@ -22,7 +22,7 @@
     // "alwaysStrict": true /* Parse in strict mode and emit "use strict" for each source file. */,
 
     /* Additional Checks */
-    "noUnusedLocals": true /* Report errors on unused locals. */,
+    "noUnusedLocals": false /* Report errors on unused locals. */,
```

Comment:
Temporary only until https://github.com/dethcrypto/TypeChain/pull/543 is fixed and we can increase the patch version of TypeChain.